shared/router-v2/tab-bar/container.js

```diff
@@ -18,6 +18,7 @@ const mapStateToProps = state => ({
   _badgeNumbers: state.notifications.navBadges,
   fullname: TrackerConstants.getDetails(state, state.config.username).fullname || '',
   isWalletsNew: state.chat2.isWalletsNew,
+  uploading: state.fs.uploads.syncingPaths.size > 0 || state.fs.uploads.writingToJournal.size > 0,
```

Comment:
FWIW I prefer `.count()` to `.size` -- if the immutable sequence is lazy, size doesn't force evaluation and returns undefined, whereas `.count()` does.